flake.nix

```
@@ -0,0 +1,80 @@
+{
+  inputs = {
+    rust-overlay = {
+      url = "github:oxalica/rust-overlay";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.flake-utils.follows = "flake-utils";
+    };
+    cargo2nix = {
+      url = "github:cargo2nix/cargo2nix/release-0.11.0";
+      inputs.rust-overlay.follows = "rust-overlay";
+    };
+    flake-utils.follows = "cargo2nix/flake-utils";
+    nixpkgs.follows = "cargo2nix/nixpkgs";
+  };
+
+  outputs = inputs: with inputs; # pass through all inputs and bring them into scope
+
+    # Build the output set for each default system and map system sets into
+    # attributes, resulting in paths such as:
+    # nix build .#packages.x86_64-linux.<name>
+    flake-utils.lib.eachDefaultSystem (system:
```

```suggestion
    flake-utils.lib.eachSystem [ "x86_64-linux" "aarch64-linux" ] (system:
```

Aurae can't be compiled on non-linux (yet?) and setting up cross compiling was an absolutely massive pain that I didn't manage to get working, so we can shelve that for now. But, this probably shouldn't claim to build on darwin in that case :)